aion/web/models/user.ex

```diff
@@ -18,7 +20,26 @@ defmodule Aion.User do
   """
   def changeset(struct, params \\ %{}) do
     struct
-    |> cast(params, [:name, :email, :encrypted_password])
-    |> validate_required([:name, :email, :encrypted_password])
+    |> cast(params, [:name, :email, :password])
+    |> validate_required([:name, :email, :password])
+    |> validate_length(:email, min: 3, max: 30)
```

Comment:
We could move constants like this to a separate module or just out of this function and not hardcode them. (especially the latter one shouldn't be a lot of work, you could deal with that using the `module @attributes` which are conventional way of dealing with module constants in Elixir.

Reply:
This is out of scope of this PR.

Reply:
True